bundles/AdminBundle/Resources/public/js/pimcore/object/tags/input.js

```diff
@@ -124,9 +129,12 @@ pimcore.object.tags.input = Class.create(pimcore.object.tags.abstract, {
 
 
     getLayoutShow: function () {
-
         this.component = this.getLayoutEdit();
-        this.component.setReadOnly(true);
+        if (this.fieldConfig.showCharCount) {
+            this.component.items.items[0].setReadOnly(true);
```

Comment:
I would prefer an approach similar to

https://github.com/pimcore/pimcore/blob/857b22beef2503abbac7f51b0479d4a1b9235982/bundles/AdminBundle/Resources/public/js/pimcore/object/tags/localizedfields.js#L99

instead of exploting implementation details ;-)